src/hooks/queries/useIsApprovalRequired.ts

```diff
@@ -15,16 +12,14 @@ type UseIsApprovalRequired = {
   amountCryptoBaseUnit: string
 }
 
+const usdtAssetId: AssetId = 'eip155:1/erc20:0xdac17f958d2ee523a2206206994597c13d831ec7'
```

Comment:
Nitpick on naming, should this be `usdtOnEthereumAssetId`?

Reply:
Conventionally, we never use any chain terminology for mainnet flavours! Note I've moved this to caip in https://github.com/shapeshift/web/pull/7670/files#diff-07cb24655c483f16cbe3aee6ae57352da77b488cda993f70567297dc1e1fe308R28, happy to change the terminology across the board there if we find it easier to understand.

Reply:
Yeah when you put it that way it makes complete sense, happy with this as-is 🤝